arkouda/strings.py

```diff
@@ -1421,15 +1421,15 @@ def save_parquet(self, prefix_path : str, dataset : str='strings_array',
         if mode.lower() in 'truncate':
```

Comment:
After discussing with @bmcdonald3, a new issue has been created to address this so globally so that the code stays in sync for HDF5 and Parquet for the time being. Issue #1230 has been added to address this.